css/at-rules/media.json

```diff
@@ -264,6 +264,61 @@
               "deprecated": false
             }
           }
+        },
+        "any-hover": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/any-hover",
+            "description": "<code>any-hover</code> media feature",
+            "support": {
+              "webview_android": {
+                "version_added": false
```

Comment:
I did [a little digging](https://www.chromestatus.com/features/6460705494532096) and it looks like Android Webview, Chrome, and Chrome for Android all shipped this in version 41. Would you mind updating all three values?